llvm/test/CodeGen/AMDGPU/vectorize-buffer-fat-pointer.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: opt -S -mtriple=amdgcn-- -passes=load-store-vectorizer < %s | FileCheck -check-prefix=OPT %s
```

Comment:
Should I rename the file? Left it as is for now to see actual changes more easily.

Reply:
I think it's fine. It's still a kind of buffer fat pointer in a sense.